packages/ra-ui-materialui/src/list/ListGuesser.tsx

```diff
@@ -79,4 +82,49 @@ ${inferredChild.getRepresentation()}
     return <ListView {...props}>{inferredChild}</ListView>;
 };
 
-ListViewGuesser.propTypes = ListView.propTypes;
+ListViewGuesser.propTypes = {
+    // @ts-ignore-line
```

Comment:
why did you need to copy the props? Can't you use Omit instead, which would make the intention clearer?

Reply:
I had a really weird compilation error and haven't found another way to fix it

Reply:
Fixed